README.md

```diff
@@ -78,9 +79,15 @@ The following parameters can be used to rename CSV-generated output columns as f
 - `--metadata_6_header`: names the first metadata_6 column header
 - `--metadata_7_header`: names the first metadata_7 column header
 - `--metadata_8_header`: names the first metadata_8 column header
+- `--earliest_header`: names the earliest date column header and related output columns
```

Comment:
Updated this missing parameter for the earliest transformation, as I noticed it was missing.